co2eq/purchase/index.js

```diff
@@ -147,6 +149,7 @@ export function carbonEmissions(activity) {
           footprint = eurAmount * 1186 / 1000; // Taxi bill
           break;
         case TRANSPORTATION_MODE_TRAIN:
+        case TRANSPORTATION_MODE_PUBLIC_TRANSPORT:
```

Comment:
Is this accurate enough for an estimation? Can't open original source any longer, but I'm wondering since trains generally have much lower footprint than fx buses, right?

Reply:
they do but I discussed with @martincollignon and it's the best number we have right now. We should improve it though.